app/mockServices/mockData/services.ts

```diff
@@ -1,16 +1,16 @@
 export const servicesData = {
-  "wildfly": {
+  "test-serviceclass-java": {
```

Comment:
Arrgg...my apologies!  I meant to remove 'service-view.classes.ts'!  It has been replaced with 'src/services/catalog.service.ts'